### PR TITLE
fix private messages sorting

### PR DESCRIPTION
**Pull request by albertoalaejos**
## Problem
The private messages are not in a chronological classification.
Private messages are sorted based on last updated time, but the private messages inbox view show the creation time, which does not always match the updated time.

## Solution
Show the last updated time instead of creation time.

## Issue tracker
https://getopensocial.atlassian.net/browse/TB-2919

## How to test
- [ ] Go to /user/inbox
- [ ] Make sure that private messages are sorted in the correct order.

### CODE REVIEW

File: themes/socialbase/templates/private_message/private-message-thread.html.twig [hunk 1]
```diff
@@ -32,7 +32,9 @@
         </div>
         <div class="media-body message__inbox-body">
           {{ content.membernames }} <span class="read-indicator"></span>
-          {{ content.last_message }}
+          <div class="message__updated">
+            {{ content.updated }} {{ 'ago' }}
```
Comment (jaapjan) [inline themes/socialbase/templates/private_message/private-message-thread.html.twig]: @albertoalaejos should be translatable 😛

**Comment by jaapjan** [comment]
I saw that there was an (unrelated) failure in Travis install_update test which is solved in 8.x-7.x already so merged that into this branch. :crossed_fingers:

File: themes/socialbase/templates/private_message/private-message-thread.html.twig [hunk 2]
```diff
@@ -32,7 +32,9 @@
         </div>
         <div class="media-body message__inbox-body">
           {{ content.membernames }} <span class="read-indicator"></span>
-          {{ content.last_message }}
```
Comment (bramtenhove) [inline themes/socialbase/templates/private_message/private-message-thread.html.twig]: You're removing the last message here. How come?

**Comment by bramtenhove** [comment]
It turns out that we thought the issue we saw with private messages ordering had to be fixed in the distro. This is not the case, I'll close the PR.

**Pull request closed without merging**